app/Lib/BulkLookup.php

```diff
@@ -39,6 +39,8 @@ public static function retrieveByCallsignOrEmail(array $people) : array
                     'status' => $person->status,
                     'email' => $person->email,
                     'last_worked' => last(Timesheet::findYears($person->id, Timesheet::YEARS_WORKED)),
+                    'vintage' => $person->vintage > 0,
+                    'years_worked' => count(Timesheet::findYears($person->id, Timesheet::YEARS_WORKED)),
```

Comment:
Timesheet::findYears() is called twice here. The best thing is to avoid a redundant SQL query and grab the result once, and then use that for the last_worked (e.g., last($years)) & years_worked (e.g. count($years)) values.

Reply:
Ah, good call. https://github.com/burningmantech/ranger-clubhouse-api/pull/1294